test/infrastructure/docker/docker/util.go

```diff
@@ -82,7 +82,7 @@ func List(filters container.FilterBuilder) ([]*types.Node, error) {
 
 func list(visit func(string, *types.Node), filters container.FilterBuilder) error {
 	ctx := context.TODO()
-	containerRuntime, err := container.NewDockerClient()
+	containerRuntime, err := container.NewRuntimeClient(ctx)
```

Comment:
We should make the context flow down the chain, otherwise this method wont work with providers different that docker